api/src/main/java/com/pip/chatbot/dao/JokeDao.java

```diff
@@ -22,10 +23,25 @@ public JokeDao(JooqConfiguration jooqConfiguration) {
         this.dslContext = jooqConfiguration.configuration();
     }
 
-    public List<Joke> getAll(){
+    public List<Joke> getAll() {
         Result<Record> result = dslContext.select().from(JOKE).fetch();
         return result.into(Joke.class);
     }
 
+    public Joke getRandomJoke(Integer value) {
```

Comment:
Użyjmy into Optional, żeby dać jawnie do zrozumienia, że rekordu może nie być. Ta metody powinna się nazywać getById, albo getOfId, bo tak naprawdę ona to robi. A jeżeli ma zwracać coś losowego, to powinniśmy to robić np. w taki sposób
```
SELECT column FROM table
ORDER BY RANDOM()
LIMIT 1
```